src/cargo/core/compiler/fingerprint/mod.rs

```diff
@@ -2102,13 +2271,14 @@ pub struct RustcDepInfo {
 struct EncodedDepInfo {
     files: Vec<(DepInfoPathType, PathBuf)>,
     env: Vec<(String, Option<String>)>,
+    checksum: Vec<(DepInfoPathType, PathBuf, u64, String)>,
```

Comment:
(not a blocker) 
I feel like `files` and `checksum` should eventually merge into one field.

Reply:
They are now the same field